src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -1,52 +1,17 @@
 <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
   <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    {% if path == "health-care" or path == "drupal/health-care" %}
-      <li>
-        <a href="/">
-          Home
-        </a>
-      </li>
-      <li>
-        <a href="/{{path}}" aria-current="page">
-          Health Care
-        </a>
-      </li>
-    {% else %}
-      {% assign numCrumbs = entityUrl.breadcrumb | size %}
-      {% assign crumbCount = 1 %}
-      {% for crumb in entityUrl.breadcrumb %}
-        {% if crumbCount == numCrumbs %}
-        <li>
-          <a href="/{{path}}" aria-current="page">
-            {{crumb.text}}
-          </a>
-        </li>
-        {% else %}
-          {% if crumb.text == "Health care" %}
-            {% assign crumbPath = "/health-care" %}
-          {% else %}
-            {% assign crumbPath = crumb.url.path %}
-          {% endif %}
-          {% if crumb.text != "Get benefits" and crumb.text != "Manage benefits" and crumb.text != "More resources" %}
-        <li>
-          {% if crumbPath != empty %}
-          <a href="{{crumbPath}}"
-            {% if crumb.url.path == "/" %}
-            onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });"
-            {% elsif crumb.text == lastCrumb %}
-            aria-current="page"
+    {% for crumb in entityUrl.breadcrumb %}
+        {% if forloop.last == true %}
+            <li><a aria-current="page" href="{{ crumb.url.path }}">{{ crumb.text }}</a></li>
```

Comment:
I think this should be `href="/{{path}}"` like deleted line 20.

Reply:
Doing that results in this: 
<img width="1003" alt="Screen Shot 2019-04-16 at 9 49 36 AM" src="https://user-images.githubusercontent.com/3157339/56224579-11352080-602d-11e9-925a-8fcfb979923c.png">

Reply:
Misread this comment, sorry. Change is made in latest commit.